src/browser/components/TabBar.jsx

```diff
@@ -29,37 +34,53 @@ export default class TabBar extends React.Component { // need "this"
       let badgeDiv;
       if (sessionExpired) {
         badgeDiv = (
-          <div className='TabBar-badge TabBar-badge-nomention'/>
+          <div className='TabBar-expired'/>
         );
       } else if (mentionCount !== 0) {
         badgeDiv = (
           <div className='TabBar-badge'>
             {mentionCount}
           </div>
         );
+      } else if (unreadCount !== 0) {
+        badgeDiv = (
+          <div className='TabBar-dot'/>
+        );
       }
-      const id = 'teamTabItem' + index;
 
-      // draggable=false is a workaround for https://github.com/mattermost/desktop/issues/667
-      // It would obstruct https://github.com/mattermost/desktop/issues/478
-      return (
+      const id = 'teamTabItem' + index;
```

Comment:
```suggestion
      const id = `teamTabItem${index}`;
```
this is optional, just for consistency